client/src/pages/User/UserProfile.js

```diff
@@ -0,0 +1,89 @@
+import React, {useState}from "react";
+import Button from "@material-ui/core/Button";
+import FileUploaderDialog from '../../component/Uploader/FileUploaderDialog';
+import "./styles.css";
+import Snackbar from '@material-ui/core/Snackbar';
+import MuiAlert from '@material-ui/lab/Alert';
+
+function Alert(props) {
+  return <MuiAlert elevation={6} variant="filled" {...props} />;
+}
+
+export default function MaxWidthDialog() {
+  const [open, setOpen] = useState(false);
+  const [loading, setLoading] = useState(false);
+  const [displaySnack, setDisplaySnack] = useState(false);
+  const [displaySnackError, setDisplaySnackError] = useState(false);
+  const [selectedFile, setSelectedFile] = useState();
+
+  const handleClickOpen = () => {
+    setSelectedFile(null);
+    setLoading(false);
+    setOpen(true);
+  };
+
+  const handleDrop = (files) => {
+    setSelectedFile(files[0]);
+  };
+
+
+  const handleClose = () => {
+    setOpen(false);
+  };
+
+  const handleSubmission = () => {
+    setLoading(true);
+    const formData = new FormData();
+    formData.append("file", selectedFile);
+    fetch("/api/file-upload", {
+      method: "POST",
+      body: formData,
+    })
+      .then((response) => response.json())
+      .then((result) => {
+        setDisplaySnack(true);
+        setDisplaySnackError(false);
```

Comment:
these could be combined into one function that takes in whether it's an error or not the and the message to display. So we only need one Snackbar

Reply:
good point. thx